src/quo/components/share/share_qr_code/view.cljs

```diff
@@ -1,46 +1,243 @@
-      {:active-opacity   1
-       :underlay-color   colors/neutral-80-opa-1-blur
-       :background-color :transparent
-       :on-press         url-on-press
-       :on-long-press    url-on-long-press
-       :style            style/profile-address-content-container}
-      [text/text
-       {:style           style/profile-address-content
-        :size            :paragraph-1
-        :weight          :medium
-        :ellipsize-mode  :middle
-        :number-of-lines 1}
-       qr-url]]]
-    [rn/view {:style style/share-button-container}
-     [button/button
-      {:icon-only?          true
-       :type                :grey
-       :background          :blur
-       :size                32
-       :accessibility-label :share-profile
-       :on-press            share-on-press}
-      :i/share]]]])
+  "Receives the following properties:
+   - type:                :profile | :wallet-legacy | :wallet-multichain
+   - qr-image-uri:        Image source value.
+   - qr-data:             Text to show below the QR code.
+   - on-text-press:       Callback for the `qr-data` text.
+   - on-text-long-press:  Callback for the `qr-data` text.
+   - on-share-press:      Callback for the share button.
+   - customization-color: Custom color for the QR code component.
+   - unblur-on-android?:  [Android only] disables blur for this component.
+
+   Depending on the `type`, different properties are accepted:
+   `:profile`
+     - full-name:       User full name.
+     - profile-picture: map ({:source image-source}) or any image source.
+   `:wallet-legacy`
+     - emoji:               Emoji in a string to show in the QR code.
+     - on-info-press:       Callback for the info icon.
+     - on-legacy-press:     Callback for the legacy tab.
+     - on-multichain-press: Callback for the multichain tab.
+   `:wallet-multichain`
+     - networks:            A vector of network names as keywords (`[:ethereum, :my-net, ...]`).
+     - on-settings-press:   Callback for the settings button.
+     - emoji:               Emoji in a string to show in the QR code.
+     - on-info-press:       Callback for the info icon.
+     - on-legacy-press:     Callback for the legacy tab.
+     - on-multichain-press: Callback for the multichain tab."
+  [{:keys [unblur-on-android?] :as props}]
+  (reagent/with-let [component-width     (reagent/atom nil)
+                     container-component (if (and platform/android? unblur-on-android?)
+                                           [rn/view {:background-color style/overlay-color}]
+                                           [blur/view
+                                            {:blur-radius      20
+                                             :blur-amount      20
+                                             :blur-type        :transparent
+                                             :overlay-color    style/overlay-color
+                                             :background-color style/overlay-color}])]
```

Comment:
about the `unblur-on-android?` param.

The blur implementation looks good on Android and iOS, but when we want to use this component on top of another blur view on Android, it looks bad :(

So I'm providing a boolean, we should set `unblur-on-android? true` when the blur implementation is having problems.

Reply:
This should probably be a comment in the file

Reply:
Will add it :+1: